src/components/modals/OperatorStakingModal.tsx

```diff
@@ -1,27 +1,21 @@
 import { AoGatewayWithAddress, ARIOToken, mARIOToken } from '@ar.io/sdk/web';
-import { Label, Radio, RadioGroup } from '@headlessui/react';
 import { EAY_TOOLTIP_FORMULA, EAY_TOOLTIP_TEXT } from '@src/constants';
 import useBalances from '@src/hooks/useBalances';
 import useGateways from '@src/hooks/useGateways';
 import useProtocolBalance from '@src/hooks/useProtocolBalance';
 import { useGlobalState } from '@src/store';
-import { WithdrawalType } from '@src/types';
 import { formatAddress, formatPercentage, formatWithCommas } from '@src/utils';
 import { calculateOperatorRewards } from '@src/utils/rewards';
 import { MathJax } from 'better-react-mathjax';
-import { useCallback, useEffect, useState, useMemo } from 'react';
+import { useCallback, useEffect, useMemo, useState } from 'react';
 import Button, { ButtonType } from '../Button';
 import LabelValueRow from '../LabelValueRow';
 import Tooltip from '../Tooltip';
 import ErrorMessageIcon from '../forms/ErrorMessageIcon';
-import {
-  validateIOAmount as validateARIOAmount,
-  validateOperatorWithdrawAmount,
-} from '../forms/validation';
-import { CircleCheckIcon, CircleIcon, InfoIcon } from '../icons';
+import { validateIOAmount as validateARIOAmount } from '../forms/validation';
```

Comment:
why not rename the actual function?

Reply:
was part of refactoring and didn't catch that auto did that, will update now